hibernate-core/src/main/java/org/hibernate/metamodel/internal/AbstractEmbeddableRepresentationStrategy.java

```diff
@@ -37,7 +38,7 @@ public AbstractEmbeddableRepresentationStrategy(
 		this.embeddableJavaTypeDescriptor = embeddableJavaTypeDescriptor;
 
 		this.propertyAccesses = new PropertyAccess[ propertySpan ];
-		this.attributeNameToPositionMap = CollectionHelper.concurrentMap( propertySpan );
+		this.attributeNameToPositionMap = new ConcurrentHashMap<>( propertySpan );
```

Comment:
No need to use the util method in CollectionHelper for `ConcurrentHashMap` constructor is enough for it will compute the size internally.